### PR TITLE
feat(order): Introduced Ordering of Icon #231

### PR DESCRIPTION
**Pull request by mannyrivera2010**
Hud Ordering of Icons

Please Review
@blynch11p 
@lsemesky

### CODE REVIEW

File: app/js/components/header/index.jsx [hunk 1]
```diff
@@ -71,7 +71,8 @@ var Header = React.createClass({
 
                     <div className="navbar-right">
                         <ul className="nav navbar-nav">
-                            <UserNotificationDropdown updateHud={LibraryActions.fetchLibrary} moreNotifications={this.showNotifications}/>
+                            <UserNotificationDropdown updateHud={LibraryActions.
```
Comment (blynch11p) [inline app/js/components/header/index.jsx]: What happened here? 'fectchLibrary' was replaced with a newline.

Comment (mannyrivera2010) [inline app/js/components/header/index.jsx]: Not sure what happened, I fixed it in  31a3004

**Comment by blynch11p** [comment]
:+1:

**Comment by lsemesky** [comment]
:+1:

**Pull request merged**